components/basecomponents/StoryText.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useRef, MutableRefObject } from 'react';
+import oPernikoveChaloupce from '../../data/translations/cs/pohadka_pernikovachaloupka.json';
+import oDvanactiMesickach from '../../data/translations/cs/pohadka_mesicky.json';
+
+interface StoryTextProps {
+  audio: HTMLAudioElement | null;
+  languageText: string;
+  languagePlay: string;
+  id: string;
+  onPlaying: (playing: boolean) => void;
+}
+
+interface StoryPhrase {
+  main: string;
+  uk: string;
+  start_cs: number;
```

Comment:
'start_main' ? Try to avoid unnecessary references to a specific language variant